src/components/Toast/ErrorToast.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link"
+import { FC } from "react"
+import { toast } from "react-hot-toast"
+
+import { Toast, TransactionToastComponentProps } from "@/components/Toast"
+
+import { FortIconWarning } from "@/icons"
+
+export const ErrorToast: FC<TransactionToastComponentProps> = ({
+  message,
+  txHash,
+  ...toastProps
+}) => {
+  return (
+    <Toast
+      className="flex items-center bg-red-500 text-white"
```

Comment:
I think we should not be centering vertically here. The icon + close button should always be in the same place (i.e. aligned at the top of the toast).